### PR TITLE
Add support for Elasticsearch plugins

### PR DESCRIPTION
**Pull request by UserNotFound**
Plugins can be installed by setting the env as follows:

    ES_PLUGINS="plugin1 plugin2 plugin3"

Tested on ES >= 2.0

### CODE REVIEW

File: test-plugin.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    fi
+    sleep 1
+  done
+
+  echo "${1} did not get installed"
+  docker logs "$DB_CONTAINER"
+  return 1
+}
+
+if dpkg --compare-versions "$ES_VERSION" lt 2; then
```
Comment (krallin) [inline test-plugin.sh]: Note that these scripts normally run on the Docker "client" side, which might be your local OSX workstation, etc.  

dpkg may not be installed there (we can get it easily on OSX, though, and we don't have anyone using CentOS / Fedora / RHEL right now), so I don't think this is a problem, but I thought I'd flag this for you!

File: test-plugin.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  -e ES_PLUGINS="$PLUGINS"  \
+  "$IMG"
+
+for PLUGIN in $PLUGINS; do
+  wait_for_plugin "$PLUGIN"
+done
+
+echo "Destroying"
+docker stop "$DB_CONTAINER"
+docker rm "$DB_CONTAINER"
```
Comment (krallin) [inline test-plugin.sh]: Just FYI, you can skip those since the `EXIT` trap will run on completion 😄

File: bin/elasticsearch-wrapper [hunk 3]
```diff
@@ -32,6 +32,18 @@ if [[ -n "${ES_HEAP_SIZE:-}" ]]; then
   fi
 fi
 
+# Install any user specified plugins
+
+if [[ -n "${ES_PLUGINS:-}" ]]; then
+  for PLUGIN in $ES_PLUGINS; do
+    if [[ -d "/elasticsearch/plugins/${PLUGIN}" ]]; then
```
Comment (UserNotFound) [inline bin/elasticsearch-wrapper]: That's probably a good long-term idea, we could even make it a 'plugin-function-wrapper', as I duplicated some logic from it in the tests in order to check installed plugins.

File: test-plugin.sh (hunk 1 above)
Comment (UserNotFound) [inline test-plugin.sh]: Yeah, I intentionally use an Ubuntu VM so that I am testing in a very similar environment to Travis-CI.

**Pull request merged**